tiledb/sm/misc/types.h

```diff
@@ -150,12 +152,21 @@ class Range {
 
   /** Returns the start as a string view. */
   std::string_view start_str() const {
-    return std::string_view((const char*)start(), start_size());
+    if (range_start_size_ == 0) {
+      return std::string_view(nullptr, 0);
+    }
+    return std::string_view((const char*)range_.data(), range_start_size_);
```

Comment:
@dhoke4tdb We should convert to static_cast here.

Reply:
As a matter of concise style, it's sufficient style to say `return {nullptr,0}` and omit the class name. For `string_view` itself, it has a default constructor that uses those values, so `return {}` works and is as short as possible.

Agree about `static_cast`.